examples/minimal/packages/client-phaser/src/layers/network/contractComponents.ts

```diff
@@ -1,24 +0,0 @@
-/* Autogenerated file. Do not edit manually. */
```

Comment:
this is an unused file and was likely leftover from copy+pasting phaser things (the phaser template removed this file a while back)